index.js

```diff
@@ -18,7 +20,17 @@ github.authenticate({
     token: process.env.FORKED_TOKEN,
 })
 
-const packageJson = shell.cat('package.json')
+const args = process.argv.slice(2)
+if (args.indexOf('-h') !== -1 || args.indexOf('--help') !== -1) {
+  console.log(`  Easy-peasy usage:
+    1. Navigate to directory of dependency you want to fork.
+    2. run \`fork\`
+
+  Lemon-squeezy usage:
+    1. From your project’s root directory, run \`fork dependencyName\`.`)
```

Comment:
My comment was put here to address the API, not the logic itself. I'd rather have an API where I put in the file path to where I want to fork something, than to have the library go try and find what I am looking for. As a user I think that would make me feel safer, and makes the API less magical.

So rather than say `$ fork fbjs` and have forked go off in my file system looking for something named `fbjs`, if I said `$ fork node_modules/fbjs` I feel safe knowing it's only going to fork the module I am pointing it towards.